applications/nrf_desktop/src/modules/Kconfig.bas

```diff
@@ -7,9 +7,11 @@
 menu "BLE BAS Service"
 
 config DESKTOP_BAS_ENABLE
-	bool
+	bool "Enable GATT Battery Service"
+	depends on BT
```

Comment:
shouldn't this configuration select/imply the ```BT_BAS``` option?

Reply:
Nope. We do not use the mentioned implementation. nRF Desktop implements BAS on its own using GATT Service API.